File: JavaScript/01.JavaScript-Basic-February-2021/06.conditionalStatementsAdvanced-Exercise/07.hotelRoom.js
function hotelRoom(input) {
    let month = input[0];
    let nights = Number(input[1]);

    let totalApartment = 0;
    let totalStudio = 0;
    if (month == "May" || month == "October") {
        if (nights > 7 && nights <= 14) {
            totalStudio = nights * 50 * 0.95;
            totalApartment = nights * 65;
        } else if (nights > 14) {
            totalStudio = nights * 50 * 0.70;
            totalApartment = nights * 65 * 0.9;
        } else if (nights <= 7) {
            totalStudio = nights * 50;
            totalApartment = nights * 65;
        }

    } else if (month == "June" || month == "September") {
        if (nights > 14) {
            totalStudio = nights * 75.2 * 0.80;
            totalApartment = nights * 68.7 * 0.9;
        } else if (nights <= 14) {
            totalStudio = nights * 75.2;
            totalApartment = nights * 68.7;
        }

    } else if (month == "July" || month == "August") {
        if (nights > 14) {
            totalStudio = nights * 76;
            totalApartment = nights * 77 * 0.9;
        } else if (nights <= 14) {
            totalStudio = nights * 76;
            totalApartment = nights * 77;
        }
    }

    console.log(`Apartment: ${totalApartment.toFixed(2)} lv.`);
    console.log(`Studio: ${totalStudio.toFixed(2)} lv.`);
}

hotelRoom(["May", "15"]);
hotelRoom(["June", "14"]);

File: Go/01.Go-Basic-February-2022/06.conditionalStatementAdvanced-Exercise/07.hotelRoom.go
// 50/100
package main

import "fmt"

func main() {
	var month string
	fmt.Scanln(&month)

	var count int
	fmt.Scanln(&count)

	var studioPrice float32
	var apartmentPrice float32

	if month == "May" || month == "October" && count > 7 {
		studioPrice = 50 * 0.95
		apartmentPrice = 65 * 0.9

	} else if month == "May" || month == "October" && count > 14 {
		studioPrice = 50 * 0.7
		apartmentPrice = 65 * 0.9

	} else if month == "June" || month == "September" && count > 14 {
		studioPrice = 75.2 * 0.8
		apartmentPrice = 68.7 * 0.9
	} else if month == "July" || month == "August" && count <= 14 {
		studioPrice = 76
		apartmentPrice = 77
	} else if month == "July" || month == "August" && count > 14 {
		studioPrice = 76
		apartmentPrice = 77 * 0.9
	}

	var totalStudio float32 = studioPrice * float32(count)
	var totalApartment float32 = apartmentPrice * float32(count)

	fmt.Printf("Apartment: %.2f lv.", totalApartment)
	fmt.Println()
	fmt.Printf("Studio: %.2f lv.", totalStudio)
}
